### PR TITLE
Master Stats

### PR DESCRIPTION
**Pull request by thatch45**
### What does this PR do?
Adds some stats to the master, so that the workers send out events with stats data for a given interval

### CODE REVIEW

File: conf/master [hunk 1]
```diff
@@ -297,6 +297,11 @@
 #batch_safe_limit: 100
 #batch_safe_size: 8
 
+# Master stats enables stats events to be fired from the master at close
+# to the defined interval
+#master_stats: False
```
Comment (mirceaulinic) [inline conf/master]: That's a very nice feature. Can we also document these options? https://docs.saltstack.com/en/latest/ref/configuration/master.html Thanks!

**Comment by thatch45** [comment]
I added a few tweeks and some docs

**Comment by thatch45** [comment]
Assuming tests pas this should be good to go

**Comment by thatch45** [comment]
@cachedout all tests pass, anything else or can I merge away?

**Comment by thatch45** [comment]
W00t! I think this is going to be very helpful

**Pull request merged**